application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -54,12 +59,29 @@ private boolean messageHasNoMediaAttached(Message message) {
                 && !message.getContentRaw().contains("http");
     }
 
-    private RestAction<Message> dmUser(Message message) {
+    private MessageEmbed originalMessageEmbed(Message message) {
```

Comment:
weird method name. i also dont quite get why it exists in the first place

Reply:
This existed previously so I didn't Intend to change it  like do I remove it or rename it now?

Reply:
it did not exist as a seperate method originally. previously it was a variable name and in that context the name made sense. but not as seperate method like that.

Reply:
this didn't existed previously, there was no method which returned the message embed and only that

Reply:
Original dmUser had a embed it was common to ping and dm so I made a function out of it

Reply:
Yes ok le me move it also